packages/medusa/src/loaders/index.ts

```diff
@@ -124,19 +125,19 @@ export default async ({
 
   container.register({ manager: asValue(dbConnection.manager) })
 
-  const stratActivity = Logger.activity("Initializing strategies")
+  const stratActivity = Logger.activity("Initializing strategies\n")
   track("STRATEGIES_INIT_STARTED")
   strategiesLoader({ container, configModule, isTest })
   const stratAct = Logger.success(stratActivity, "Strategies initialized") || {}
   track("STRATEGIES_INIT_COMPLETED", { duration: stratAct.duration })
 
-  const servicesActivity = Logger.activity("Initializing services")
+  const servicesActivity = Logger.activity("Initializing services\n")
```

Comment:
question(non-blocking): why are we adding the line breaks?

Without being certain I think this might mess things up on windows machines and in production logging

Reply:
They were added in an attempt to make our server log more readable on initial startup. See image below for reference.

That being said, it's not critical so I'll leave it for a different PR (if this is something we want tot perfect)

![CleanShot 2022-11-17 at 10 48 53](https://user-images.githubusercontent.com/59018053/202413457-726e23dd-a350-46f7-b3f1-708db8e04b6c.png)